src/main/java/online/shop/model/ShoppingCard.java

```diff
@@ -0,0 +1,38 @@
+package online.shop.model;
+
+import java.util.List;
+
+public class ShoppingCard {
+    private Long id;
+    private List<Product> products;
+    private Long userId;
+
+    public ShoppingCard(List<Product> products, long userId) {
+        this.products = products;
```

Comment:
Same here. I think out mentors have been commenting on this saying that we will not receive a List of products as a parameter, and we should just initialize the List<Product> products in the constructor.